graders/python/python_autograder/pl_result.py

```diff
@@ -13,20 +13,13 @@ class PLTestResult(unittest.TestResult):
     """
 
     error_message = (
-        "The grading code failed -- sorry about that!\n\n"
-        "This may be an issue with your code.\nIf so, you can "
-        "take a look at the traceback below to help debug.\n"
-        "If you believe this is an issue with the grading code,\n"
-        "please notify the course staff.\n\n"
-        "The error traceback is below:\n"
+        "There was an error while grading your code.\n\n"
+        "Review the question text to ensure your code matches\nthe expected requirements, such as variable names,\nfunction names, and parameters.\n\n"
+        "Look at the traceback below to help debug your code:\n"
     )
-
-    major_error_message = (
-        "The grading code was not able to run.\n"
-        "Please notify the course staff "
-        "and include this entire message,\n"
-        "including the traceback below.\n\n"
-        "The error traceback is:\n"
+    grader_error_message = (
+        "There was a grader error while grading your code.\n\n"
```

Comment:
Continuing to bikeshed, this reads slightly better to me?

```suggestion
        "The grader encountered an error while grading your code.\n\n"
```